### PR TITLE
refactor: Rename the `Welcome` screen to `TryOrInstall`

### PR DESCRIPTION
**Pull request by spydon**

### CODE REVIEW

File: packages/ubuntu_bootstrap/lib/l10n/ubuntu_bootstrap_ar.arb [hunk 1]
```diff
@@ -15,33 +15,33 @@
...
+  "tryDescription": "يمكنك تجربة {RELEASE} بدون إجراء اي تغييرات لجهازك.",
+  "@tryDescription": {
     "type": "text",
     "placeholders": {
       "RELEASE": {}
     }
   },
-  "welcomeInstallOption": "ثبت {RELEASE}",
+  "installOption": "ثبت {RELEASE}",
   "@welcomeInstallOption": {
```
Comment (d-loose) [inline packages/ubuntu_bootstrap/lib/l10n/ubuntu_bootstrap_ar.arb:37]: should be changed to `@installOption` here and probably in all other arb files

Comment (spydon) [reply packages/ubuntu_bootstrap/lib/l10n/ubuntu_bootstrap_ar.arb:37]: Good catch!

Comment (spydon) [reply packages/ubuntu_bootstrap/lib/l10n/ubuntu_bootstrap_ar.arb]: Fixed

**Review by d-loose — COMMENTED** [review]
There's still some inconsistency in the arb files. Running `grep welcome packages/ubuntu_bootstrap/lib/l10n/*.arb` shows some strings that haven't been renamed

**Comment by spydon** [comment]
> There's still some inconsistency in the arb files. Running `grep welcome packages/ubuntu_bootstrap/lib/l10n/*.arb` shows some strings that haven't been renamed

Haha how annoying, apparently Android Studio hasn't properly indexed those files on my computer so those some instances doesn't show up in the search and replace box.

**Review by d-loose — APPROVED** [review]
Thanks, LGTM :+1:

**Pull request merged**